test/integration/targets/user/tasks/main.yml

```diff
@@ -186,6 +186,47 @@
   when: ansible_facts.system != 'Darwin'
 
 
+# https://github.com/ansible/ansible/issues/68775
+- block:
+    - name: add maximum expire date for password
+      user:
+        name: ansibulluser
+        password_expire_max: 10
+      register: pass_max_1_0
+
+    - name: again add maximum expire date for password
+      user:
+        name: ansibulluser
+        password_expire_max: 10
+      register: pass_max_1_1
+
+    - name: validate result for maximum expire date
+      assert:
+        that:
+          - pass_max_1_0 is changed
+          - pass_max_1_1 is not changed
+
+    - name: add minimum expire date for password
+      user:
+        name: ansibulluser
+        password_expire_min: 5
+      register: pass_min_2_0
+
+    - name: again add minimum expire date for password
+      user:
+        name: ansibulluser
+        password_expire_min: 5
+      register: pass_min_2_1
+
+
+    - name: validate result for minimum expire date
+      assert:
+        that:
+          - pass_min_2_0 is changed
+          - pass_min_2_1 is not changed
+  when: ansible_facts.system == 'Debian'
```

Comment:
This is a great start. You can use the `getent` module to look at the shadow file and verify it got the correct values.
```yaml
- name: Get shadow data for ansibulluser
  getent:
    database: shadow
    key: ansibulluser
  
- name: Ensure password expiration was set properly
  assert:
    that:
       - getent_shadow[2] == '5'
       - getent_shadow[3] == '10'
```

Reply:
ok working on it.